lib/announce/adapters/shoryuken_adapter.rb

```diff
@@ -33,6 +33,7 @@ def subscribe(worker_class, subject, actions, options)
         def register_class(worker_class)
           if active_job?
             Class.new(AnnounceWorker).tap do |jc|
+              jc.class_eval("shoryuken_options body_parser: :json, auto_delete: true")
```

Comment:
guys I need to specify this directly because somehow the inheritance of the AnnounceWorker Class does not doing it. Feels weird though, but maybe you face this issue at some point.

Let me know what you think about it